content/en/tools/digital-currencies/viewing-the-gui-settings.md

```diff
@@ -0,0 +1,39 @@
+---
+date: '2023-03-21'
+lastmod: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: "Viewing the GUI Settings"
+    weight: 300
+    parent: digital-currencies
+    identifier: digital-currencies-changing-settings
+    description: "Digital Currencies documentation describing how to view and change the settings of the GUI"
+title: "Viewing the GUI Settings"
+---
+
+This topic describes how to view and change settings associated with the GUI, such as its theme.
+
+* Click the **Settings** button (![](images/setting-buttons.png)).
+
+  The following page is displayed:
+  
+  {{< 
+      figure
+	  src="images/settings-page.png"
+      width=80%
+	  figcaption="Settings Page"
+	  alt="Settings Page"
+  >}}
+     
+The page displays the following panels:
+
+* **Settings**: This panel displays the following configurable settings:
+
+  * **Dark app theme:** Select this option to change the GUI to have a dark  background.
+  * **Show flow tracker:** Select this option to display the flow tracker button on the right-hand side of the GUI. 
+  * **Hide flow tracker on click away:** Select this option to automatically hide the flow tracker if the user clicks anywhere outside it.
+
+* **VNode Details:** This panel displays the X.500 and Short Hash values specified for the virtual node (as described in [Installing the Digital Currencies Demo]({{< relref "installing-the-dc-demo.md" >}}). These values are not changeable in the GUI.
```

Comment:
```suggestion
* **VNode Details:** This panel displays the X.500 and Short Hash values specified for the virtual node (as described in [Installing the Digital Currencies Demo]({{< relref "installing-the-dc-demo.md" >}})). These values are not changeable in the GUI.
```

I'm not sure if you need a space in-between the link and the )

Reply:
Got rid of the first round bracket and replaced with a comma.